apps/calculation/calculation.h

```diff
@@ -95,6 +95,7 @@ friend CalculationStore;
   // Additional Information
   AdditionalInformationType additionalInformationType(Poincare::Context * context);
 private:
+  static constexpr int maxWidth = 314;
```

Comment:
This should be something like Ion::Display::Width - something?

Reply:
I changed "314" to "Ion::Display::Width - 6" in my latest commit